### ISSUE TITLE
Error: Program "make" not found in PATH

### ISSUE THREAD

**Issue by KX31** [clough42/electronic-leadscrew#251]
Hello all,

I am trying to load the code to the launchpad but i am experiencing some error trying to build the code.
I started with CCS V12 , then CCS V10 and V9; they all have the same issues. 

Am i missing something in the install or setting?

Here is the error message:

{time} **** Clean-only build of configuration Release for project els-f280049c ****
make clean 
Cannot run program "make": Launching failed

Error: Program "make" not found in PATH

PATH=[
... ... ...
]

{time} Build Failed. 1 errors, 0 warnings. (took 48ms)

**** Build of configuration Debug for project els-f280049c ****

Nothing to build for project els-f280049c

Thanks for your help,

**Comment by clough42** [comment]
The firmware is written for v9.  If you're still seeing the error, it's
likely your system is still trying to use V12 or some newer version.  I'd
probably start by making sure everything is removed from your machine and
reinstalling v9 from scratch.

James

**Comment by KX31** [comment]
Thanks for the help, i managed to un-install V10 & V12 properly and re-install V9.
It build properly, but now it error to load onto the board.

I went back over the video and spec sheet and something is wired:

The board select properly:
![Screenshot 2023-03-30 165119](https://user-images.githubusercontent.com/88797127/228992761-a946f726-10a3-4146-af2b-0ce446229ba7.png)

But it doesn't have the XDS degugger in the selection:
![Screenshot 2023-03-30 165100](https://user-images.githubusercontent.com/88797127/228992833-0b27dcbe-0fc6-4611-bdab-4302025500c9.png)

And explicitly state the XTS is not installed:
![Screenshot 2023-03-30 165142](https://user-images.githubusercontent.com/88797127/228992879-ad1f0734-c921-4ef2-93b0-4b73c84df0cd.png)

Im missing something (that im sure is simple) but im stuck.

Ho, and here is the error code on loading: 
![Screenshot 2023-03-30 172007](https://user-images.githubusercontent.com/88797127/228992969-214a74d5-545d-4963-a546-c9fa912e9204.png)

I tried installing V9 a few times, and always get the same selection on the install procedure (always missing the XDS)
Is there a dedicated install needed for it?
